themes/bootstrap3/templates/ajax/resolverLink.phtml

```diff
@@ -1,7 +1,5 @@
 <?php if (!empty($link['href'])): ?>
-  <a href="<?=$this->escapeHtmlAttr($link['href'])?>" title="<?=$this->transEscAttr($link['service_type'] ?? '')?>"<?=!empty($link['access']) ? ' class="access-' . $link['access'] . '"':''?>>
-    <?=$this->escapeHtml($link['title'] ?? '')?>
-  </a>
+  <a href="<?=$this->escapeHtmlAttr($link['href'])?>" title="<?=$this->transEscAttr($link['service_type'] ?? '')?>"<?=!empty($link['access']) ? ' class="access-' . $link['access'] . '"':''?>><?=$this->escapeHtml($link['title'] ?? '')?></a>
```

Comment:
@crhallberg, does this change serve any purpose? It looks like the markup is the same, but it's been made less readable by being mashed into a single line. Was the extra whitespace causing a problem that needed to be addressed, or was this an accidental change? If it's intentional, we should add a comment explaining why so that it doesn't get reformatted in the future. If it's not intentional, we should revert it to eliminate an unnecessary change.